### ISSUE TITLE
Work Package Detail Revamp - Changes Tab

### ISSUE THREAD

**Issue by RChandler234** [Northeastern-Electric-Racing/FinishLine#1124]
Blocked by #1121 

Populate the Changes tab with content (should have the same data as the changes section did on the old wp detail page, just without the pageblocks, see the project detail page or this PR: https://github.com/Northeastern-Electric-Racing/FinishLine/pull/1112, for an example of how to do that):

This PR should merge into a feature branch called "#1097-wp-detail-revamp" instead of develop.

![image](https://user-images.githubusercontent.com/29521172/234752539-7eeb4b32-b811-4ea7-8e1f-dead869a6ed0.png)

Epic: #1097
